src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -51,6 +51,11 @@ export type TravelDetailsMapScreenParams = {
   mode?: AnyMode;
   subMode?: AnySubMode;
   mapFilter?: MapFilterType;
+  // An event to notify that the screen close button was pressed and can take some actions before it is closed,
+  // it differs from `onPressBack` which is called/overridden by the parent call-in Dashboard_TravelDetailsMapScreen.tsx
+  // By doing this, it prevents replacing `onPressBack` with a different action that prevents calling the navigation
+  // at the top view or avoids overflow in the stack.
+  onScreenClose?: () => void;
```

Comment:
Shouldn't have functions in screen params types, since functions are not serialisable. 
https://reactnavigation.org/docs/5.x/troubleshooting#i-get-the-warning-non-serializable-values-were-found-in-the-navigation-state

Reply:
I removed it, and instead, I used a different approach.